### PR TITLE
test(contrib): upload-kktix-ticket-csv-to-bigquery.py: add test cases for 2019

### PR DESCRIPTION
**Pull request by tai271828**
## Types of changes

- **Other (contrib)**

## Description

Add test cases and the corresponding test data for 2019

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [ ] Update the documentation if necessary

## Steps to Test This Pull Request

`python -m unittest upload-kktix-ticket-csv-to-bigquery.py`

## Expected behavior

All tests pass.

### CODE REVIEW

File: contrib/upload-kktix-ticket-csv-to-bigquery.py [hunk 1]
```diff
@@ -322,5 +322,63 @@ def test_hash_email(self):
         )
 
 
+class Test2019Ticket(unittest.TestCase):
+    """python -m unittest upload-kktix-ticket-csv-to-bigquery.py"""
```
Comment (david30907d) [inline contrib/upload-kktix-ticket-csv-to-bigquery.py:326]: 💯  LGTM, my comment is the same as <https://github.com/pycontw/PyCon-ETL/pull/36#pullrequestreview-688404781>

Comment (tai271828) [reply contrib/upload-kktix-ticket-csv-to-bigquery.py:326]: Thanks for the review!

File: contrib/upload-kktix-ticket-csv-to-bigquery.py [hunk 2]
```diff
@@ -322,5 +322,63 @@ def test_hash_email(self):
         )
 
 
+class Test2019Ticket(unittest.TestCase):
```
Comment (uranusjr) [inline contrib/upload-kktix-ticket-csv-to-bigquery.py:325]: Are we using pytest or not? If we are, this does not need to use `unittest.TestCase`. If we are not, tests should use `assertEqual()` etc. functions, not bare `assert`.

Comment (tai271828) [reply contrib/upload-kktix-ticket-csv-to-bigquery.py:325]: Good catch! Should be assertEqual +1. It's the unittest of standard lib.

Comment (tai271828) [reply contrib/upload-kktix-ticket-csv-to-bigquery.py:325]: Updated and re-based this pull request. Please refer to this commit https://github.com/pycontw/PyCon-ETL/pull/34/commits/2bf312b26a878cddc68e4e0ec05957ba4deefb0c implemented by following your suggestion.

**Comment by tai271828** [comment]
This pull request is ready to review again. Thanks!

File: contrib/upload-kktix-ticket-csv-to-bigquery.py [hunk 3]
```diff
@@ -322,5 +322,61 @@ def test_hash_email(self):
...
+    def setUpClass(cls):
+        cls.df = pd.read_csv("./data/corporate-attendees-2019.csv")
+        cls.sanitized_df = sanitize_column_names(cls.df)
+
+    def test_column_number(self):
+        self.assertEqual(len(self.sanitized_df.columns), 18)
+
+    def test_column_title_content(self):
+        for column in self.sanitized_df.columns:
+            self.assertTrue(column in self.CANONICAL_COLUMN_NAMES_2019)
```
Comment (uranusjr) [inline contrib/upload-kktix-ticket-csv-to-bigquery.py:359]:
```suggestion
            self.assertIn(column, self.CANONICAL_COLUMN_NAMES_2019)
```

**Review by uranusjr — CHANGES_REQUESTED** [review]

File: contrib/upload-kktix-ticket-csv-to-bigquery.py (hunk 3 above)
Comment (tai271828) [reply contrib/upload-kktix-ticket-csv-to-bigquery.py]: Good catch (again)! Thank you for your wonderful heads-up!

**Comment by tai271828** [comment]
Updated this pull request based on the latest suggestions. When the CI passes, this pull request is ready to review again. Thanks!

**Review by uranusjr — APPROVED** [review]

**Pull request merged**